client/tests/stubs.js

```diff
@@ -20,6 +20,7 @@ global.wagtailConfig = {
     SHORT_DATE_FORMAT: 'DD/MM/YYYY',
   },
   STRINGS: {
+    EDIT: 'Edit',
```

Comment:
It would be nice for this to be in the same order as they are defined in `localization.py` to make it easier to compare.